docs/sources/datasources/pyroscope/profilng-tracing-intro.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Profiling and tracing synergies
+menuTitle: Profiling and tracing
+description: Learning about how profiling and tracing work together.
+weight: 250
+aliases:
+  - ../introduction/traces-to-profiles/
+  - ../introduction/profile-tracing/
```

Comment:
Does a new page need aliases?

Reply:
This was a copy of an old page, so no, not here. This page was based on the one from Pyroscope. Good catch!